dali/core/dynlink_cuda.cc

```diff
@@ -67,10 +66,10 @@ bool cuInitChecked() {
       return true;
 
   // set symbol loader for this library
-#if !LINK_LIBCUDA_ENABLED
+#if !LINK_DRIVER_ENABLED
   CudaSetSymbolLoader(LoadSymbol);
 #endif
-  static CUresult res = cuInit(0);
+  CUresult res = cuInit(0);
```

Comment:
is this in purpose? Do we want to run cuInit every time?

Reply:
Reverted. Done